presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+* Improved performance of sum and average aggregations on decimals.
+* Support for LocationProvider supplied paths in Iceberg connector.
+* Column comments now available in JDBC connector.
+* Hidden columns ``file_modified_time`` and ``file_size`` added to Hive connector.
+* Partition lease support added to Hive connector.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix the Aggregation IF to FILTER rule by not unwrapping the IF expression inside the Aggregation. The rule now rewrites ``AGG(IF(condition, expr))`` to ``AGG(IF(condition, expr)) FILTER (WHERE condition)``.
+* Fix marking coordinator active so it is only done when the required minimum resource managers and required minimum coordinators are available in a cluster. Required resource managers can be configured by ``cluster.required-resource-managers-active`` (default value: 1, disaggregated coordinator only). Required coordinators can be configured by ``cluster.required-coordinators-active`` (default value: 1).
```

Comment:
This is very long and not easy to understand. Can we have a short description of what user will get, with a reference to the issue / PR for more details?

Reply:
@swapsmagic can you help with shorten this and link it to the PR?

Reply:
* Fix marking coordinator active when required resource manager (``cluster.required-resource-managers-active``, default: 1) and coordinators (``cluster.required-coordinators-active``,  default: 1) are available in a cluster. 
Does that seems reasonable?